src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -108,10 +107,9 @@ const ProjectSelector = React.createClass({
     return label;
   },
 
-  getRawLink(project) {
+  getProjectPath(project) {
     let org = this.props.organization;
-    let urlPrefix = ConfigStore.get('urlPrefix');
-    return urlPrefix + '/' + org.slug + '/' + project.slug + '/';
+    return '/' + org.slug + '/' + project.slug + '/';
```

Comment:
this breaks outside of react if you have a subpath installation of Sentry

Its not something thats tested or officially supported, but enough people do it we should try to not break i